adjust.go

```diff
@@ -171,6 +173,12 @@ func (f *File) adjustSingleRowDimensions(sheet string, r *xlsxRow, num, offset i
 	for i, col := range r.C {
 		colName, _, _ := SplitCellName(col.R)
 		r.C[i].R, _ = JoinCellName(colName, r.R)
+	}
+	return nil
+}
+
+func (f *File) adjustSingleRowFormulas(sheet string, r *xlsxRow, num, offset int, si bool) error {
```

Comment:
This function never return error type values, and the parameters `num` and `si` no longer required.